examples/architectures/VGG8.py

```diff
@@ -115,8 +114,8 @@ def _fc_layer_with_dropout(bottom, name, size,
 
     with tf.variable_scope(name) as scope:
         n1 = bottom.get_shape()[1].value
-        stddev = (2/n1)**0.5
-        #stddev = 0.04
+        stddev = (2 / n1)**0.5
+        # stddev = 0.04
```

Comment:
This should be removed (or a comment should be added when one wants to have it - then a flag should be there to enable / disable it without changing the code)